DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/lacp/test_lacp_blocked_port_stp_rstp.py

```diff
@@ -0,0 +1,165 @@
+import asyncio
+import pytest
+
+from dent_os_testbed.lib.ip.ip_link import IpLink
+from dent_os_testbed.lib.mstpctl.mstpctl import Mstpctl
+from dent_os_testbed.utils.test_utils.tgen_utils import tgen_utils_get_dent_devices_with_tgen
+
+pytestmark = [pytest.mark.suite_functional_devlink,
+              pytest.mark.asyncio,
+              pytest.mark.usefixtures('cleanup_tgen', 'cleanup_bonds', 'cleanup_bridges', 'enable_mstpd')]
```

Comment:
You don't need cleanup_tgen fixture here since you don't set up any traffic there